build-ps/build-binary.sh

```diff
@@ -258,6 +258,7 @@ fi
         -DDOWNLOAD_BOOST=1 \
 	-DFORCE_INSOURCE_BUILD=1 \
 	-DWITH_LIBEVENT=bundled \
+	-DWITH_ZSTD=bundled \
```

Comment:
Indentation of this line and the previous 2 seems off

Reply:
fixed